doc/docs/master/concepts/advanced-concepts/deferred_processing.md

```diff
@@ -284,6 +284,40 @@ When more than one is specified, a branch repoint will be triggered when any of
 the conditions is met. To guarantee that they all must be met, add
 --trigger-all.
 
+## Embed Triggers in Pipelines
+
+Triggers can also be specified in the pipeline spec and automatically created
+when the pipeline is created. For example, this is the edges pipeline from our
+our OpenCV demo modified to only trigger when there is a MB of new images:
```

Comment:
a MB ->  1 Megabyte

so we're consistent with prior paragraphs.

Reply:
```suggestion
our OpenCV demo modified to only trigger when there is a 1 Megabyte of new images:
```